agr_literature_service/lit_processing/update_pubmed_papers.py

```diff
@@ -134,7 +135,13 @@ def update_data(mod, pmids, md5dict=None):  # noqa: C901
     fw.write("Generating json files...\n")
     log.info("Generating json files...")
 
-    generate_json(pmids_all, [])
+    not_found_xml_list = []
+    generate_json(pmids_all, [], not_found_xml_list)
+
+    if newly_added_pmids:
+        for pmid in newly_added_pmids:
+            if pmid in not_found_xml_list:
+                not_found_xml_list.remove(pmid)
```

Comment:
We could use a set() instead of a list and use the .discard() function

Reply:
I see. I will fix this.

Reply:
Fixed.

Reply:
thanks @sweng66 !